tools/devtool

```diff
@@ -550,6 +564,50 @@ cmd_fmt() {
 }
 
 
+# Prepare a Firecracker release by updating the version, crate dependencies
+# and credits.
+# Example: `devtool release v0.42.0`
+#
+cmd_release() {
```

Comment:
Since this is a new command, it should also have a detailed description showing up under `devtool --help`.